junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -262,20 +262,25 @@ default Store getStore() {
 	Store getStore(Namespace namespace);
 
 	/**
-	 * {@code Store} provides methods for extensions to save and retrieve data.
+	 * Provides methods for extensions to save and retrieve data.
```

Comment:
Again, what was here contradicts the JavaDoc practices, so I was cleaning up while making other changes.

If you want to require a different style , I suggest you document it.